src/engraving/dom/utils.cpp

```diff
@@ -1738,10 +1738,10 @@ bool segmentsAreInDifferentRepeatSegments(const Segment* firstSeg, const Segment
         return false;
     }
 
-    const MeasureBase* firstMasterMeasureBase = master->measure(firstMeasure->index());
+    const MeasureBase* firstMasterMeasureBase = master->tick2measureBase(firstMeasure->tick());
```

Comment:
Doesn't this potentially cause different results when there are multiple MeasureBases with the same tick? (for example, multiple frames in succession?)
And shouldn't (some of) the other cases use <code>tick2measure<I>Base</I></code> as well?

Reply:
Ah yes - this is a typo. They should all be `tick2measure`. None of these methods are dealing with anything other than `Measure`

Reply:
If they are only dealing with Measure, then we don't need conversions from Measure to MeasureBase and back to Measure again